generators/server/templates/gradle/profile_prod.gradle.ejs

```diff
@@ -23,7 +23,7 @@ apply plugin: 'com.moowork.node'
 <%_ } _%>
 
 dependencies {
-
+    testCompile "com.h2database:h2"
```

Comment:
@PierreBesson @joewhite101 : I think it should be added only when h2 is selected for dev database. Need to be excluded for other database like MongoDB, Cassandra, etc.

Reply:
@pascalgrimaud Yeah you are correct. May be a few days before I get a chance to look at it but I will create a new PR sometime soon.

Reply:
thanks @joewhite101 but it should be OK :)
PR already done this morning: https://github.com/jhipster/generator-jhipster/pull/8212